prism.el

```diff
@@ -1073,6 +1067,13 @@ Receives one argument, a color name or hex RGB string."
   :type 'function
   :set #'prism-customize-set)
 
+(defcustom prism-parens-fn
+  (lambda (color) color)
```

Comment:
This would seem to change the default behavior, no?

Anyway, if these changes are really necessary, please make them separately from this PR.  I prefer that each PR contain the minimum changes necessary to implement its purpose, because it makes it easier for me to understand and review the code.

Reply:
Oops, yes, I should have used the default given here. My intention with the PR was generally to enable the rainbows theme, but my title did not describe that well, that's my fault, sorry, I haven't done many PRs. 

In this case, without the ability to customize the parens-fn, users could not get the visual effect displayed in the gif. The more general reason I disabled the strings/parens/comment differentiation is because it just ends up looking way too noisy to me.